### ISSUE TITLE
Toggle sorting of self.classes in ConfusionMatrix

### ISSUE THREAD

**Issue by lewiuberg** [sepandhaghighi/pycm#412]
#### Description
I want to add my classes instead of the default `[0, 1, 2]` and in a particular order `["normal", "lgg", "hgg"]`. However, these get sorted. It would be nice to toggle this on/off.

#### Steps/Code to Reproduce
```python
cm = ConfusionMatrix(
    y_true, y_pred, digit=5
)  # bug with classes=cfg.dataset.classes

cm.relabel(
    mapping={
        0: cfg.dataset.classes[0],
        1: cfg.dataset.classes[1],
        2: cfg.dataset.classes[2],
    }
)
```

#### Expected Behavior
```python
cm.relabel(
    mapping={
        0: cfg.dataset.classes[0],
        1: cfg.dataset.classes[1],
        2: cfg.dataset.classes[2],
    },
    sort=False
)
```

```zsh
pycm.ConfusionMatrix(classes: ['normal', 'lgg', 'hgg'])
```

#### Actual Behavior
```zsh
pycm.ConfusionMatrix(classes: ['hgg', 'lgg', 'normal'])
```

#### Operating System
macOS Monterey V 12.0.1

#### Python Version
3.9

#### PyCM Version (Use : `pycm.__version__`)
3.3

**Comment by lewiuberg** [comment]
Here is my temporary solution for my particular problem.

```python
def replace_text_in_file(file_path, text_to_replace, replacement_text):
    with open(file_path, "r") as f:
        content = f.read()
    new_content = content.replace(text_to_replace, replacement_text)
    with open(file_path, "w") as f:
        f.write(new_content)


def remove_sort_from_pycm_classes():
    import os
    import site

    from pycm import ConfusionMatrix

    SITE_PACKAGES_PATH = site.getsitepackages()
    PYCM_PATH = os.path.join(SITE_PACKAGES_PATH[0], "pycm")
    PYCM_OBJ_FILE = os.path.join(PYCM_PATH, "pycm_obj.py")
    print(PYCM_OBJ_FILE)

    replace_text_in_file(
        PYCM_OBJ_FILE,
        "self.classes = sorted(list(mapping.values()))",
        "self.classes = list(mapping.values())",
    )


remove_sort_from_pycm_classes()
```

**Comment by sepandhaghighi** [comment]
Good point!
We will work on it in a future release.
@sadrasabouri

**Comment by lewiuberg** [comment]
Great!

If you like I can give it a go later this week :)

**Comment by sadrasabouri** [comment]
> Great!
> 
> If you like I can give it a go later this week :)

Yes that would be totally good. So I'm assigning you to it.
Feel free and thanks for your efforts. :100:

**Comment by lewiuberg** [comment]
Great! My first actual assignment :) I'll do it right after some pressing tasks. Have a good night!

**Comment by sepandhaghighi** [comment]
> Great! My first actual assignment :) I'll do it right after some pressing tasks. Have a good night!

Very well! 🥇 
Before starting please take a look at our [Contributing guidelines](https://github.com/sepandhaghighi/pycm/blob/master/.github/CONTRIBUTING.md)

**Comment by lewiuberg** [comment]
Hi! Some really pressing issues came in. I have not forgotten, but I had to postpone all other tasks for a while. Sorry:)

**Comment by sadrasabouri** [comment]
> Hi! Some really pressing issues came in. I have not forgotten, but I had to postpone all other tasks for a while. Sorry:)

No problem. Take your time. There is no rush.
